.circleci/config.yml

```
@@ -1,7 +1,71 @@
 version: 2.1
 orbs:
   rust: circleci/rust@1.6.0
+executors:
+  docker-publisher:
+    docker:
+      - image: cimg/rust:1.71.1
 jobs:
+  docker-build-glados-web-and-publish:
+    resource_class: xlarge
+    executor: docker-publisher
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          no_output_timeout: 30m
+          command: docker build -f glados-web/Dockerfile -t glados-web:latest --build-arg GIT_HASH=$CIRCLE_SHA1 .
+      - run:
+          name: Publish docker image to Docker Hub
+          command: |
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+            docker push glados-web:latest
```

```suggestion
            docker push portalnetwork/glados-web:latest
```

Doesn't this need the `portalnetwork/` prefix?

---

good catch

---

fixed